pulsar-client/src/main/java/org/apache/pulsar/client/impl/GeoReplicationProducerImpl.java

```diff
@@ -108,6 +108,15 @@ private void ackReceivedReplicatedMsg(ClientCnx cnx, OpSendMsg op, long sourceLI
         if (pendingLId != null && pendingEId != null
                 && (pendingLId < lastPersistedSourceLedgerId || (pendingLId.longValue() == lastPersistedSourceLedgerId
                   && pendingEId.longValue() <= lastPersistedSourceEntryId))) {
+            if (MessageImpl.SchemaState.Broken.equals(op.msg.getSchemaState())) {
+                log.error("[{}] [{}] Replication due to incompatible schem and the replicator will be stuck by producer"
+                                + " queue size limitation."
+                                + " Latest published entry {}:{}, Entry who has broken schema: {}:{},"
+                                + " latest persisted source entry: {}:{}, pending queue size: {}.",
```

Comment:
Would it be possible to improve the clarity of this error message? 

* "Replication due to incompatible schem" is not very clear.
  * What does that mean exactly?
* " and the replicator will be stuck by producer queue size limitation" 
  * Is there a way to make this more understandable and possibly [actionable](https://developers.google.com/tech-writing/error-messages/show-fix)?

Reply:
Improved the log